### PR TITLE
Fixes #RHIROS-1402 - Fixed payload creation for kruize updateResult API

### PR DESCRIPTION
**Pull request by patilsuraj767**

### CODE REVIEW

File: internal/services/report_processor.go [hunk 1]
```diff
@@ -238,3 +242,19 @@ func ProcessReport(msg *kafka.Message) {
 		}
 	}
 }
+
+func sliceK8sObjectToChunks(k8s_objects []kruizePayload.UpdateResult) [][]kruizePayload.UpdateResult {
```
[the remark this answers is not in the document]
Comment (saltgen) [reply internal/services/report_processor.go:246]: @patilsuraj767 Additionally, I didn't get the reason for moving this function to current file.

**Review by saltgen — CHANGES_REQUESTED** [review]

File: internal/services/report_processor.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (patilsuraj767) [reply internal/services/report_processor.go:246]: I was getting linting issue because of cyclic imports. So I moved the function out of utils file.

**Review by saltgen — APPROVED** [review]

File: internal/services/report_processor.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (patilsuraj767) [reply internal/services/report_processor.go]: I had a look into `Dependency Inversion Principle` not sure how it fits in for this case but thanks for unblocking the PR. I will request you if possible you can implement it and create the new PR, will be happy to learn new things.

**Comment by patilsuraj767** [comment]
PR checks are failing because of some unknown issue. This has been reported to the QE team. 
I have tested the PR in ephemeral environment manually and everything look good. Hence merging the PR.

**Pull request merged**